lib/src/registry/registration.dart

```diff
@@ -0,0 +1,42 @@
+library pointycastle.src.registry.impl;
+
+import 'package:pointycastle/digests/md2.dart';
+import 'package:pointycastle/digests/md4.dart';
+import 'package:pointycastle/digests/md5.dart';
+import 'package:pointycastle/digests/ripemd128.dart';
+import 'package:pointycastle/digests/ripemd160.dart';
+import 'package:pointycastle/digests/ripemd256.dart';
+import 'package:pointycastle/digests/ripemd320.dart';
+import 'package:pointycastle/digests/sha1.dart';
+import 'package:pointycastle/digests/sha224.dart';
+import 'package:pointycastle/digests/sha256.dart';
+import 'package:pointycastle/digests/sha3.dart';
+import 'package:pointycastle/digests/sha384.dart';
+import 'package:pointycastle/digests/sha512.dart';
+import 'package:pointycastle/digests/sha512t.dart';
+import 'package:pointycastle/digests/tiger.dart';
+import 'package:pointycastle/digests/whirlpool.dart';
+import 'package:pointycastle/src/registry/registry.dart';
+
+void registerFactories(FactoryRegistry registry) {
```

Comment:
this is actual registration method.
this is still in progress, but does this make sense?